src/code/models/codap-connect.coffee

```diff
@@ -121,7 +124,7 @@ module.exports = class CodapConnect
                 singleCase: 'run'
                 pluralCase: 'runs'
               attrs: [
-                name: tr '~CODAP.SIMULATION.EXPERIMENT'
+                name: @defaultExperimentName
```

Comment:
Yes.  This is a bit confusing.

The use case is that an author sets up a CODAP document with sage, and some other CODAP plugins. They then make copies of this CODAP document for several languages.

The author didn't run any simulations yet, so regardless of their language setting, the experiment number column has the default label.

When an i18 user collects experiment data for the first time, we then rename the column from the default.

Reply:
A couple of thoughts:
1. A comment in the code would be a good idea.
2. What this is essentially saying is that time of creation is not a good time to decide the appropriate language for a document, so we should wait until the first experiment is run to make that determination.
3. Fathom handled this by always saving/restoring the canonical name for localizable attributes, functions, etc. and then always using localized display names generated dynamically at run time so that the Fathom-provided parts of documents were always displayed in the user's current language rather than being locked into a particular language. (The user content did not have this dynamic quality, of course.)

Reply:
The code changes look good.